### ISSUE TITLE
Test Case: ( sjs ) Projects - Paging inconsistencies

### ISSUE THREAD

**Issue by leobardoramon** [gosha1128/NYUAMS#545]
When going from Table View to Radio View (aka List View to Detail View), the Paging does not follow the selected project.  For example, when a project from the <b>bottom</b> of the list is selected in Table View, and then the view is switched to Radio View, the Order Sub-table is shown to the right of the Radio table, but the actual projects shown are from the <b>top</b> of the list.

Server: Production
URL: lgsdb.nyu.edu
user: admin
pass: password

Area: Projects page

![screen shot 2015-07-22 at 4 15 03 pm](https://cloud.githubusercontent.com/assets/2165796/8836170/1c24f2c8-308d-11e5-9462-d61065d2a2af.png)
![screen shot 2015-07-22 at 4 15 09 pm](https://cloud.githubusercontent.com/assets/2165796/8836171/1c26caf8-308d-11e5-99c2-2d0a653932df.png)

Video Link:  https://drive.google.com/open?id=0B_9JEK6SVk1bRzF2djVWSnRwQzg

**Comment by barryross** [comment]
@leobardoramon 

So the issue really seems to be with "Smart Paging".  Note in the video, the pagination issue only occurs when using that feature before selected a fixed page size.  Furthermore, using the smart paging first, then using fixed paging, prevents the user from selecting a project in "fixed" mode.  

https://www.dropbox.com/s/avxoy42hcgf1mc1/545-paging.mov?dl=0

I commented out much of the "smart paging" feature for "Projects", and it actually resolved  these issues, albeit there is a longer list.  

This solution work for you guys? 

https://www.dropbox.com/s/h4b51r7leimz4gd/545-possible-solution.mov?dl=0

**Comment by leobardoramon** [comment]
@buck0land can we just set the paging to "page size 25" as a default.

This should fix that issue

**Comment by leobardoramon** [comment]
@barryross , can we please proceed with the most recent solution?

**Comment by barryross** [comment]
@leobardoramon yup, sorry for the delay.  I have implemented this on the `orders` and `projects` page (because they share the same paginator).  Can you please let me know if this works for you? 

Relevant commit https://github.com/codetodesign/NYUAMS/commit/f758374a567720fdc3697779e971e2107944c634

Video Demo
https://www.dropbox.com/s/avxoy42hcgf1mc1/545-paging.mov?dl=0
